src/json_merger.py

```diff
@@ -69,7 +71,7 @@ class ListJsonMerger(BaseJsonMerger):
 
     json_obj: list
 
-    def merge_a_list(self, the_list: list) -> list:
```

Comment:
Do you not like the `-> X`?

Reply:
You're not actually returning anything from this method, it's just setting the value of an attribute

Reply:
Well more correctly, python is defaulting to returning None as no return value was given